src/components/Lock.js

```diff
@@ -0,0 +1,44 @@
+import Auth0Lock from 'auth0-lock';
+
+let frontendURL;
+if(process.env.NODE_ENV === 'development'){
+    frontendURL = 'http://localhost:3000';
+} else {
+    frontendURL = `https://www.wellbroomed.com`
+}
```

Comment:
Would prefer something like this to handle the different environments:
`const frontendURL = process.env.REACT_APP_FRONTEND_URL || 'http://localhost:3000'`

Reply:
Done. 👍 Went ahead and tidied up the callbackUrl declaration in Auth.js as well.